ddtrace/llmobs/_integrations/litellm.py

```diff
@@ -16,3 +34,61 @@ def _set_base_span_tags(
             span.set_tag_str("litellm.request.model", model)
         if host is not None:
             span.set_tag_str("litellm.request.host", host)
+
+    def _llmobs_set_tags(
+        self,
+        span: Span,
+        args: List[Any],
+        kwargs: Dict[str, Any],
+        response: Optional[Any] = None,
+        operation: str = "",
+    ) -> None:
+        model_name = span.get_tag("litellm.request.model")
```

Comment:
Don't love being dependent on span tags. Can we either store this in the span store via `span._set_ctx_item()` or just use args/kwargs to get this again?